components/brave_sync/brave_profile_sync_service_impl.cc

```diff
@@ -1042,6 +1145,18 @@ BraveSyncService* BraveProfileSyncServiceImpl::GetSyncService() const {
 void BraveProfileSyncServiceImpl::SendSyncRecords(
     const std::string& category_name,
     RecordsListPtr records) {
+  RunWhenModelIsLoaded(
+      base::BindOnce(&BraveProfileSyncServiceImpl::SendSyncRecordsBkmLoaded,
```

Comment:
the convention here is `SendSyncRecordsImpl` and again we don't want to use abbreviations like Bkm